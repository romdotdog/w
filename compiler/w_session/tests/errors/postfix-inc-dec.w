fn main(x: u32) {
    x++;
    x--;
}