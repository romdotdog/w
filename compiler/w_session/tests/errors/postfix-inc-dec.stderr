tests/errors/postfix-inc-dec.w:2:6: error: `x++` and `x--` will not be implemented
    x++;
     ^^
tests/errors/postfix-inc-dec.w:3:6: error: `x++` and `x--` will not be implemented
    x--;
     ^^
